s3backgrounddelete/s3backgrounddelete/object_recovery_scheduler.py

```diff
@@ -219,10 +219,33 @@ def schedule_periodically(self):
         def periodic_run(scheduler):
             """Add key value to queue using scheduler."""
             #Conditionally importing ObjectRecoveryRabbitMq/ObjectRecoveryMsgbusConsumer when config setting says so.
+            from s3backgrounddelete.object_recovery_msgbus import ObjectRecoveryMsgbus
+
+            if not self.producer:
+                self.producer = ObjectRecoveryMsgbus(
+                    self.config,
+                    self.logger)
+
+            maxKeys = self.config.get_max_keys()
+            count = self.producer.get_count()
+            threshold = 0.2*maxKeys
```

Comment:
0.5 should be threshold.

Reply:
added new param "threshold" to config file, as per the discussion.